recipes/browserforge/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "browserforge" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/browserforge-{{ version }}.tar.gz
+  sha256: 7036d73fb066a4361a015b619079474c42d8b0ff415e1d874b62366de48d0b61
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python >={{ python_min }}
+    - click
+    - typing_extensions
+  run_constrained:
+    - orjson *
```

Comment:
Is this really needed? Usually one constrained by a specific version, no?

Reply:
I was curious about that one too; it's generated by Grayskull. I can remove?

Reply:
Poetry strikes again: https://github.com/daijro/browserforge/blob/main/pyproject.toml#L33
Just leave it, it is harmless and they may change to a proper pin in future releases.